src/desktop/apps/fair/routes.coffee

```diff
@@ -24,8 +26,8 @@ InfoMenu = require '../../components/info_menu/index.coffee'
   filterData = { size: 0, fair_id: fair.id, aggregations: aggregationParams }
   infoMenu = new InfoMenu(fair: fair)
   Q.all([
-    fair.fetch(cache: true)
-    infoMenu.fetch(cache: true)
+    fair.fetch(cache: true, cacheTime: DEFAULT_CACHE_TIME)
+    infoMenu.fetch(cache: true, cacheTime: DEFAULT_CACHE_TIME)
```

Comment:
We basically need to pass `cacheTime` in the `fetch` call in order to override the default cache time.